mandatory/1-fix-functions.js

```diff
@@ -10,22 +10,24 @@
 
 */
 
-function getMood() {
-  let isHappy = true;
-
+function getMood(isHappy) {
   if (isHappy) {
     return "I am happy";
   } else {
     return "I am not happy";
   }
 }
 
-function greaterThan10(num) {
-  let isBigEnough;
 
-  if (isBigEnough) {
+
```

Comment:
This looks good to me - but 2 small questions I'd like you to think about:
1. Is the `isBigEnough` variable needed? Or can you use the `num` variable directly in your if statement?
2. Do you need the `console.log`'s?